resources/definitions/weedo_x40.def.json

```diff
@@ -0,0 +1,334 @@
+            "emotiontech_petg",
+            "emotiontech_pla",
+            "emotiontech_pla_hr_870",
+            "emotiontech_pva-m",
+            "emotiontech_pva-s",
+            "emotiontech_tpu98a",
+            "eSUN_PETG_Black",
+            "eSUN_PETG_Grey",
+            "eSUN_PETG_Purple",
+            "eSUN_PLA_PRO_Black",
+            "eSUN_PLA_PRO_Grey",
+            "eSUN_PLA_PRO_Purple",
+            "eSUN_PLA_PRO_White",
+            "Vertex_Delta_ABS",
+            "Vertex_Delta_PET",
+            "Vertex_Delta_PLA",
+            "Vertex_Delta_TPU",
+            "chromatik_pla",
+            "dsm_arnitel2045_175",
+            "dsm_novamid1070_175",
+            "fabtotum_abs",
+            "fabtotum_nylon",
+            "fabtotum_pla",
+            "fabtotum_tpu",
+            "fdplast_abs_tomato",
+            "fdplast_petg_gray",
+            "fdplast_pla_olive",
+            "fiberlogy_hd_pla",
+            "filo3d_pla",
+            "filo3d_pla_green",
+            "filo3d_pla_red",
+            "imade3d_petg_green",
+            "imade3d_petg_pink",
+            "imade3d_pla_green",
+            "imade3d_pla_pink",
+            "imade3d_petg_175",
+            "imade3d_pla_175",
+            "innofill_innoflex60_175",
+            "layer_one_black_pla",
+            "layer_one_dark_gray_pla",
+            "layer_one_white_pla",
+            "octofiber_pla",
+            "polyflex_pla",
+            "polymax_pla",
+            "polyplus_pla",
+            "polywood_pla",
+            "redd_abs",
+            "redd_asa",
+            "redd_hips",
+            "redd_nylon",
+            "redd_petg",
+            "redd_pla",
+            "redd_tpe",            
+            "structur3d_dap100silicone",
+            "tizyx_abs",
+            "tizyx_flex",
+            "tizyx_petg",
+            "tizyx_pla_bois",
+            "tizyx_pla",
+            "tizyx_pva",
+            "ultimaker_abs_black",
+            "ultimaker_abs_blue",
+            "ultimaker_abs_green",
+            "ultimaker_abs_grey",
+            "ultimaker_abs_orange",
+            "ultimaker_abs_pearl-gold",
+            "ultimaker_abs_red",
+            "ultimaker_abs_silver-metallic",
+            "ultimaker_abs_white",
+            "ultimaker_abs_yellow",
+            "ultimaker_bam",
+            "ultimaker_cpe_black",
+            "ultimaker_cpe_blue",
+            "ultimaker_cpe_dark-grey",
+            "ultimaker_cpe_green",
+            "ultimaker_cpe_light-grey",
+            "ultimaker_cpe_plus_black",
+            "ultimaker_cpe_plus_transparent",
+            "ultimaker_cpe_plus_white",
+            "ultimaker_cpe_red",
+            "ultimaker_cpe_transparent",
+            "ultimaker_cpe_white",
+            "ultimaker_cpe_yellow",
+            "ultimaker_nylon_black",
+            "ultimaker_nylon_transparent",
+            "ultimaker_pc_black",
+            "ultimaker_pc_transparent",
+            "ultimaker_pc_white",

```

Comment:
You don't need to exclude all of these, they are 2.85 materials. Since the machine is 1,75, they will never show up anyway.